app/Widgets/Hero_Slider_Widget.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace Boilerplate\Theme\Widgets;
+
+use JustCoded\WP\Framework\Page_Builder\v25\Page_Builder_Widget;
+use JustCoded\WP\Framework\Web\View;
+
+/**
+ * Class Hero_Slider_Widget
+ * Custom image wordpress widget based on siteorigin widgets bundle
+ */
+class Hero_Slider_Widget extends Page_Builder_Widget {
+	/**
+	 * Hero_Slider_Widget constructor.
+	 */
+	function __construct() {
+		parent::__construct(
+			'hero-image',
+			__( 'Slider image widget' ),
+			array(
+				'description' => __( 'Slider image widget.' ),
+				'has_preview' => true,
+			),
+			array(),
+			false,
+			''
+		);
+	}
+
+	/**
+	 * Return a list of images to be shown as preview.
+	 * By default it's widget {id_base}.png with $this->name
+	 *
+	 * @return array
+	 */
+	public function get_preview_images() {
+		return array(
+			'slider-image.png' => 'Preview of the slider widget',
```

Comment:
use correct name as shown in pattern in comments

Reply:
renamed